DiagnosticsExtension/Controllers/ConnectionStringValidationController.cs

```diff
@@ -7,41 +7,46 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Net.Sockets;
 using System.Text;
 using System.Threading.Tasks;
+using System.Web;
 using System.Web.Http;
+using DiagnosticsExtension.Models;
 using DiagnosticsExtension.Models.ConnectionStringValidator;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace DiagnosticsExtension.Controllers
 {
-    /// <summary>
```

Comment:
This summary does not look correct, was it a placeholder?

Reply:
This was accidentally checked in from previous PR, I am trying to remove it in this PR.